src/test/kotlin/lotto/domain/LottoCheckerTest.kt

```diff
@@ -0,0 +1,58 @@
+package lotto.domain
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+
+class LottoCheckerTest : BehaviorSpec({
+    given("당첨번호가 ${Lotto.LOTTO_NUMBER_SIZE}개가 주어지지 않았다") {
+        val winNumbers = (Lotto.LOTTO_MIN_NUMBER until Lotto.LOTTO_MIN_NUMBER + Lotto.LOTTO_NUMBER_SIZE - 1).toList()
+        `when`("해당 당첨번호로 결과를 조회하면") {
+            then("예외가 던져진다") {
```

Comment:
벡틱 문자가 불편해 보일 수 있는데, 이 경우 대문자로 시작하는 구문을 사용하는 방법도 있습니다🙂
```suggestion
    Given("given") {
        When("when") {
            Then("then") {
            
            }
        }
    }
```

Reply:
아핫 ㅋㅋㅋ ide에서 매크로를 소문자로 설정해놔서 불편하지는 않은데 대문자로 해도 나쁘진 않겠네요 ㅋㅋㅋ